src/GitTfs/Commands/Clone.cs

```diff
@@ -139,7 +139,7 @@ public int Run(string tfsUrl, string tfsRepositoryPath, string gitRepositoryPath
             {
                 errorOccurs = true;
                 throw new GitTfsException("error: a problem occurred when trying to clone the repository. Try to solve the problem described below.\nIn any case, after, try to continue using command `git tfs "
-                    + (_fetch.BranchStrategy == BranchStrategy.All ? "branch init --all" : "fetch") + "`\n", ex);
+                    + (_fetch.BranchStrategy == BranchStrategy.All ? "--branch init --all" : "fetch") + "`\n", ex);
```

Comment:
See https://github.com/git-tfs/git-tfs/blob/master/src/GitTfs/Commands/Branch.cs#L55

It seems the right fix is (and that's what is advised in #1135 ):
```suggestion
                    + (_fetch.BranchStrategy == BranchStrategy.All ? "branch --init --all" : "fetch") + "`\n", ex);
```

Reply:
I spotted this yesterday as well after I created the pull request , but messed up when fixing it and didn't push it immeditaly. I pushed the corrected version now, which can be seen when clicking on "forced pushed" link below, which is empty. 
Sorry for the inconveniance.